src/background/portManager.ts

```diff
@@ -121,9 +121,17 @@ export async function getPidFromPort(
     isNested
   );
 
-  const stdout = execFileSync(exec.cmd, exec.args, {
-    shell: true,
-  }).toString();
+  // lsofは、ポートを使用しているプロセスが存在しない場合は
+  // エラーを返すので、エラーを無視して割り当て可能として扱う
+  let stdout: string;
+  try {
+    stdout = execFileSync(exec.cmd, exec.args, {
+      shell: true,
+    }).toString();
+  } catch (e) {
+    portLog(hostInfo.port, "Assignable; Nobody uses this port!", isNested);
+    return undefined;
+  }
```

Comment:
lsofエラー以外のときも同じとこに入るので、そのエラーハンドリングはあった方が良さそうに思いました。
が、どうやって見極めればいいかちょっと調べただけでは分かりませんでした･･･。
とりあえずFIXMEコメントを書いておくのはどうでしょう。

```suggestion
  // lsofは、ポートを使用しているプロセスが存在しない場合は
  // エラーを返すので、エラーを無視して割り当て可能として扱う
  // FIXME: lsof以外のエラーだった場合のエラーハンドリングを追加する
  let stdout: string;
  try {
    stdout = execFileSync(exec.cmd, exec.args, {
      shell: true,
    }).toString();
  } catch (e) {
    portLog(hostInfo.port, "Assignable; Nobody uses this port!", isNested);
    return undefined;
  }
```

Reply:
良いと思います！
ひとまずsuggestを取り込みますね...!